File: Kyoo/ClientApp/src/models/search-result.ts
import { Show } from "./show";
import { Episode } from "./episode";
import { People } from "./people";
import { Studio } from "./studio";
import { Genre } from "./genre";

export interface SearchResut
{
	shows: Show[];
	episodes: Episode[];
	people: People[];
	genres: Genre[];
	studios: Studio[];
}

File: Kyoo/Models/SearchResult.cs
using System.Collections.Generic;

namespace Kyoo.Models
{
    public class SearchResult
    {
        public IEnumerable<Show> shows;
        public IEnumerable<Episode> episodes;
        public IEnumerable<People> people;
        public IEnumerable<Genre> genres;
        public IEnumerable<Studio> studios;
    }
}
